map2csv/app/streamlit_app.py

```
@@ -0,0 +1,185 @@
+            M = get_2pt_affine(pts_img, pts_geo)
+            
+            if st.button("位置合わせ完了"):
+                st.session_state["homography"] = M
+                st.rerun()
+    
+    else:
+        M = st.session_state["homography"]
+        
+        try:
+            font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 20)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)